src/components/GamePage/GamePage.jsx

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+
+import BoxUi from './BoxUi';
+import TotalScoreUi from './TotalScoreUi';
+import RollDiceUi from './RollDiceUi';
+import { useState } from 'react';
+import toast from 'react-hot-toast';
+
+const ScoreContainer = styled.section`
+  display: flex;
+  justify-content: space-around;
+  margin-top: 45px;
+
+  @media (max-width: 800px) {
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 5rem;
+  }
+
+  @media (max-width: 600px) {
+    transform: scale(0.75);
+    margin-bottom: 1rem;
+  }
+`;
+
+const GamePage = () => {
+  const [shuffledNumber, setShuffledNumber] = useState(1);
+  const [selected, setSelected] = useState();
+
+  const shuffleDice = (min, max) => {
+    return Math.round(Math.random() * (max - min) + min);
+  };
+
+  const rolledDice = Number(shuffleDice(1, 6));
```

Comment:
If its parameter will not change, you can hard code it so will no need to pass params